packages/hoppscotch-common/src/components/http/Body.vue

```diff
@@ -190,4 +190,21 @@ const isContentTypeAlreadyExist = () => {
 
 // Template refs
 const tippyActions = ref<any | null>(null)
+
+// When the user clicks to change the content type
+@click="
+  () => {
+    // If the selected content type is 'none'
+    if (contentTypeItem === 'none') {
+      // Set the body.contentType to null
+      body.contentType = null
+      // And remove the 'Content-Type' header
+      headers.value = headers.value.filter(header => header.key.toLowerCase() !== 'content-type');
+    } else {
+      // Otherwise, set the body.contentType as usual
+      body.contentType = contentTypeItem
+    }
+    hide()
+  }
```

Comment:
You have included Vue template code inside the script tag. Please revise your pull request.